frontend/craco.config.js

```diff
@@ -24,6 +24,19 @@ module.exports = {
       // https://github.com/mzgoddard/hard-source-webpack-plugin
       webpackConfig.plugins.unshift(new HardSourceWebpackPlugin())
 
+      const minimizerIndex = webpackConfig.optimization.minimizer.findIndex(
+        item => item.options.terserOptions
+      )
+
+      // ⚠️ If you use Circle CI or any other environment that doesn't
+      // provide real available count of CPUs then you need to setup
+      // explicitly number of CPUs to avoid Error: Call retries were exceeded
+      // Disabling parallel when running in CircleCI
+      const cpuCount = process.env.CIRCLECI ? false : true
+      webpackConfig.optimization.minimizer[
+        minimizerIndex
+      ].options.parallel = cpuCount
```

Comment:
Previous attempts to set this to 2 for CircleCI did not work. Instead just disabling parallel :(

Reply:
You might want to leave this as a comment in the code as well since your current comment talks about explicitly setting the number of cpus, but then you're setting parallel to false instead of doing what the comment refers to.